Add example to watch multiple namespaces for istio operator (#7571)
istio/istio.io: content/en/docs/setup/install/standalone-operator/index.md

    - A service to access operator metrics
    - Necessary Istio operator RBAC rules

>     You can also pass single or multiple namespaces to watch by using `--watchedNamespaces` flag.

Reviewer: Why does just this one flag need an example? What namespaces does it watch by default?

Author: If there are no namespaces provided to watch, by [default it will watch](https://github.com/istio/istio/pull/24696/files#diff-76640f501d7b88f49ad1adbe61517bdcR5) `istio-system`.

Author: This flag needs an example because it has created confusion for users. See next comment.

    {{< text bash >}}

>     $ istioctl operator init --watchedNamespaces=istio-namespace1,istio-namespace2

Reviewer: Why is this flag not documented? https://preliminary.istio.io/latest/docs/reference/commands/istioctl/#istioctl-operator-init

Reviewer 2: i think it's because the feature is not implemented yet - see PR description.

Author: Users are confused with the term `istioNamespace` see this https://github.com/istio/istio/issues/24459  issue, if the user provides this flag, they assume that the Istio will be installed in the provided namespace. But the reality is that the `--istioNamespace` flag in `operator init` is used to watch the namespace. 

To avoid this confusion @morvencao has added `--watchedNamespaces` which clearly states that it can watch multiple namespaces, Documenting it will really help to avoid confusion. See more details https://github.com/istio/istio/pull/24696.

Reviewer 3: Thanks @shamsher31 
`--watchedNamespaces` is more clear than `--istioNamespace`,  the namespace where istio components are installed should be defined in IOP instance, should not be specified in `operator init` command.

Reviewer: This whole thing (separate namespaces for 1. running operator, 2. CRDs that operator responds to, 3. istio is installed in) seems like an overly complicated design. What happens if there are multiple CRDs in different watched namespaces with different configurations of Istio in istio-system? Why not require that 2 and 3 are the same, and watchedNamespaces would be "all namespaces"? Then you only need one flag in `istioctl operator init` to specify the namespace to run operator.

>     You can configure which namespaces the operator controller and Istio are installed in, the namespace(s) the operator watches, the installed Istio image sources and versions, and more. For example, you can pass one or more namespaces to watch using the `--watchedNamespaces` flag:

Reviewer: ```suggestion
    You can configure which namespace the operator controller is installed in, the namespace(s) the operator watches, the installed Istio image sources and versions, and more. For example, you can pass one or more namespaces to watch using the `--watchedNamespaces` flag:
```

    {{< /text >}}

>     See the available `istioctl operator init` flags to control which namespaces the operator controller and Istio are installed into and which namespaces to watch.

Reviewer: Why are you removing the part: `and the installed Istio image sources and versions.`?

It seem this whole sentence could be improved. For example:
```suggestion
    You can configure which namespaces the operator controller and Istio are installed in, the namespace(s) the operator watches, the installed Istio image sources and versions, and more. See the [`istioctl operator init` command reference](/docs/reference/commands/istioctl/#istioctl-operator-init) for details.
```

Author: This looks good. Thanks

>     You can also pass single or multiple namespaces to watch by using `--watchedNamespaces` flag.
> 
>     {{< text bash >}}
>     $ istioctl operator init --watchedNamespaces=istio-namespace1,istio-namespace2
>     {{< /text >}}
> 
>     You can configure which namespaces the operator controller and Istio are installed in, the namespace(s) the operator watches, the installed Istio image sources and versions, and more. See the [`istioctl operator init` command reference](/docs/reference/commands/istioctl/#istioctl-operator-init) for details.

Reviewer: ```suggestion
    You can configure which namespaces the operator controller and Istio are installed in, the namespace(s) the operator watches, the installed Istio image sources and versions, and more. For example, you can pass one or more namespaces to watch using the `--watchedNamespaces` flag:

    {{< text bash >}}
    $ istioctl operator init --watchedNamespaces=istio-namespace1,istio-namespace2
    {{< /text >}}

    See the [`istioctl operator init` command reference](/docs/reference/commands/istioctl/#istioctl-operator-init) for details.
```

Author: This looks good. Thanks
